From 55783283d1def69718451ce2c51dc2565762de8d Mon Sep 17 00:00:00 2001
From: David Allsopp <david.allsopp@metastack.com>
Date: Thu, 3 Aug 2017 22:59:57 +0100
Subject: [PATCH] Revert "Cudf.load_universe: set universe size to the number
 of pkgs to load"

This reverts commit 5ff068061419b25e632db67ca72f1937ce9f764e.
---
 cudf.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cudf.ml b/cudf.ml
index d82359d..2a87c4f 100644
--- a/cudf.ml
+++ b/cudf.ml
@@ -184,7 +184,7 @@ let remove_package univ id =
   end
 
 let load_universe pkgs =
-  let size = List.length pkgs in
+  let size = 2* (List.length pkgs) -1 in
   let univ = empty_universe ~size () in
   let uid = ref 0 in
   List.iter
-- 
2.12.0.windows.1

